packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -385,7 +405,8 @@ Future<int> buildAndroid(
   String outputFile: _kDefaultOutputPath,
   String target,
   String flxPath,
-  ApkKeystoreInfo keystore
+  ApkKeystoreInfo keystore,
+  bool buildForDeploy: false
```

Comment:
I probably would have used an enum here instead.  Debug and Deploy are the only current options, but Profile is likely in the longer term?